foundation_cms/static/scss/components/footer.scss

```
@@ -1,22 +1,171 @@
-footer.site-footer {
-  ul {
-    list-style: none;
+.site-footer {
+  $text-primary: #f7f7f7;
+  $bg: #0f0f0f;
+  $fg-primary: #fff;
+  $separator-color: #a3a3a3;
+  $input-padding: 1rem;
+
+  background-color: $bg;
+  color: $text-primary;
+  padding: 3rem 0;
+
+  &__legals {
+    border-top: 1px solid $separator-color;
+    padding-top: 2rem;
+    margin-top: 2rem;
+    font-size: 0.75rem;
     display: flex;
-    flex-direction: row;
-    gap: 20px;
-    margin: 0;
-    padding: 0;
+    flex-direction: column;
+    gap: 2rem;
+
+    @include breakpoint(medium up) {
```

```suggestion
    @include breakpoint(large up) {
```
let's update the breakpoint here to resolve overflowing issue